pkgs/os-specific/darwin/apple-source-releases/adv_cmds/xcode.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, appleDerivation, fetchurl, xcbuild, libcxx }:
+
+appleDerivation {
+  # disable pkill from build
+  patchPhase = ''
+    substituteInPlace adv_cmds.xcodeproj/project.pbxproj \
+      --replace "FD201DC214369B4200906237 /* pkill.c in Sources */," ""
```

Comment:
I'm not too concerned about `pkill`, honestly. For now I'd just leave it the way you have it, possibly with some breadcrumbs in a comment telling people where to find the missing headers. But tbh, since `pkill` isn't needed for bootstrap I probably wouldn't pull in the osx private SDK to the bootstrap just to support `pkill`. Perhaps eventually we can put a conditional dependency on the developer SDK (which I still need to update) and do this more cleanly with official Apple headers, and then only enable `pkill` if the condition is true.